sources/oracle/infoschema.go

```diff
@@ -101,6 +156,14 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 			switch c {
 			case "C":
 				ignored.Check = true
+			// Oracle 21c introduces a JSON datatype, before that we used to store json as VARCHAR2, CLOB, and BLOB.
+			// If column has check constraints IS JSON(check for J in constraints array as per GetConstraints function) then update src datatype to JSON
```

Comment:
we should add the json from check_constraints too

Reply:
A custom constraint "J" is defined and returned in the query result if a column has a JSON check constraint. Therefore we are using the case "J" for JSON